llumnix/entrypoints/llumnix_utils.py

```diff
@@ -182,6 +189,13 @@ def init_llumnix_components(engine_manager_args: EngineManagerArgs,
                 ray.get(llumlet.is_ready.remote())
             except ray.exceptions.RayActorError:
                 retry_manager_method_sync(engine_manager.scale_down.remote, 'scale_down', instance_ids[idx])
-    logger.info("Init Llumnix components done")
+                dead_instances.append(instance_ids[idx])
+
+    llumlets_avaliable = len(instance_ids) > len(dead_instances)
+
+    if llumlets_avaliable:
+        retry_manager_method_sync(engine_manager.scale_up.remote, 'scale_up', instance_ids, llumlets)
+        logger.info("Init Llumnix components done")
+        request_output_queue = init_request_output_queue()
```

Comment:
request_output_queue should be initialized no matter llumlets available, request_output_queue is created for api server.

Reply:
get